db/users/utils.go

```diff
@@ -0,0 +1,33 @@
+package users
+
+import (
+	"context"
+	"errors"
+	"strings"
+
+	"github.com/Scalingo/cli/config"
+	scErrors "github.com/Scalingo/go-utils/errors/v2"
+)
+
+var ErrDatabaseNotSupportUserManagement = errors.New("Error: DBMS does not support user management")
```

Comment:
**issue:** not a fan of making a constant of an error, as we never do this anywhere in our code. we often have a same base error such as "user cannot be fetched", but we do not factorize them

Reply:
> just in this repo I can see some

not really sure about that one as I never saw one in my peregrinations in all our go code. in fact, i like the idea that we centralize our errors, and then identify them more easily. though, i see this more as a consistency issue. if we do this here, we should do it every where errors are duplicated. 

this is, though, not blocking